authorization/1.6/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Jakarta Authorization 1.6"
+date: 2019-07-22T23:17:09+0200
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Authorization defines a low-level SPI for authorization modules, which are repositories of permissions 
+facilitating subject based security by determining whether a given subject has a given permission, and algorithms
+to transform security constraints for specific containers (such as Jakarta- Servlet or Enterprise Beans) into 
```

Comment:
remove dash, `s/Jakarta- Servlet/Jakarta Servlet`

Reply:
The idea was to shorten it, so it would be instead "Jakarta Servlet or Jakarta Enterprise Beans", but I'll update it. Thx

Reply:
Ah, I see what you were trying to do now.
Didn't come across like that on first read.
In this case I see no harm in being verbose.

Reply:
Okay, I updated it everywhere, thank again!